etc/taskcluster/decisionlib.py

```diff
@@ -173,6 +173,22 @@ def with_treeherder(self, category, symbol=None):
         self.treeherder_required = False  # Taken care of
         return self
 
+    def with_s3_upload_secret(self):
```

Comment:
This is Servo-specific, so I’d prefer to have it in `decision_task.py` instead. (The file separation is that `decisionlib.py` could be copy-pasted into another project’s tree, and eventually might become a proper library.)